mindjunkies/lecture/tests/test_models.py

```
@@ -1,60 +1,71 @@
-
-    def test_lecture_video_status_default(self):
-        video = baker.make(LectureVideo)
-        assert video.status == LectureVideo.PENDING
-
-    def test_lecture_video_fields_optional(self):
-        video = baker.make(LectureVideo, hls=None, thumbnail=None)
-        assert video.hls is None
-        assert video.thumbnail.name is None
+import pytest
+from django.utils.text import slugify
+from django.contrib.auth import get_user_model
```

Ruff (0.8.2)

`django.contrib.auth.get_user_model` imported but unused

Remove unused import: `django.contrib.auth.get_user_model`

(F401)